contracts/mocks/token/ERC7540Mock.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IERC20} from "../../token/ERC20/IERC20.sol";
+import {ERC20} from "../../token/ERC20/ERC20.sol";
+import {ERC7540} from "../../token/ERC20/extensions/ERC7540.sol";
+import {ERC4626} from "../../token/ERC20/extensions/ERC4626.sol";
+
+contract ERC7540Mock is ERC7540 {
+    constructor(address asset) ERC20("ERC4626Mock", "E4626M") ERC7540(IERC20(asset)) {}
```

Comment:
for some reason, it seems this is breakin, the transpiler
```suggestion
    constructor(IERC20 asset) ERC20("ERC4626Mock", "E4626M") ERC7540(asset) {}
```

Reply:
Hmm strange, i didnt encounted this error on mu local environment, i have updated it anyways.

Reply:
The transpiler is not usually run locally. That is the kind of things that shows up in CI